public/app/controls-test.js

```
@@ -0,0 +1,45 @@
+    var results = [];
+    var passed = 0;
+    var failed = 0;
+
+    function check(label, condition) {
+      var ok = Boolean(condition);
+      results.push({ label: label, ok: ok });
+      if (ok) { passed++; } else { failed++; }
+    }
+
+    // Play / transport controls
+    check('playBtn exists', document.getElementById('playBtn'));
```

ESLint

[error] 'document' is not defined.

(no-undef)